zulip_bots/zulip_bots/bots/file_uploader/file_uploader.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+from pathlib import Path
+
+class FileUploaderHanlder(object):
+    def usage(self) -> str:
+        return (
+            'This interactive bot used to upload files, images, etc to Zulip server.\n'
+            'Specify the file path you want to upload and file will be upload to Zulip server'
+        )
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
+        path = Path(message['content'].strip())
+        if not path.is_file():
+            bot_handler.send_reply(message, 'File not found')
+            return
+
+        upload = bot_handler.upload_file_from_path(str(path.resolve()))
+        if upload['result'] != 'success':
+            msg = upload['msg']
+            bot_handler.send_reply(message, 'Failed to upload file: {}'.format(msg))
+            return
+
+        uploaded_file_reply = '[{}]({})'.format(path.name, upload['uri'])
+        bot_handler.send_reply(message, uploaded_file_reply)
+
```

Comment:
There is no `help` command; while this is a very minimal bot, this might be useful to include :)

Reply:
Added `help` command